packages/documentation-site/patternfly-docs/content/design-guidelines/content/punctuation.md

```diff
@@ -17,73 +14,94 @@ Use bold text (not quotation marks) when referring to an element or text in the
+
+- "I enjoy the following hobbies: cooking, drawing, and traveling."
+- "That leads me to my favorite hobby: running."
+
+Use a **semicolon** to connect two closely related independent clauses. You can also use a semicolon instead of a comma to separate long list items for extra clarity. For example:
+
+- "I love running in the morning; it wakes me up."
+- "Every morning, I enjoy eating toast, bacon, and eggs; reading a book; and relaxing on the porch."
+
+**Note**: If you're tempted to use a semicolon in the UI, try breaking up the sentence and cutting down on your words instead. This often leads to content that is more readable and clear.
 
 ## Commas
-When a conjunction connects two independent clauses, a comma should precede it. Also put a comma before “and” if it’s the Oxford comma.
+When a conjunction connects two independent clauses, a **comma** should precede it. Also put a comma before “and” if it’s the Oxford comma. For example:
 
-Examples:
-- *I like to run, and I like to swim* — A comma is needed before “and” because “and” connects two independent clauses.
-- *I like to run and swim* — A comma is not needed before “and” because “and” does not connect two independent clauses.
-- *I like to run, swim, and hike* — The Oxford comma is included before “and.”
+- "I like to run, and I like to swim" — A comma is needed before “and” because “and” connects two independent clauses.
+- "I like to run and swim: — A comma is not needed before “and” because “and” does not connect two independent clauses.
+- "I like to run, swim, and hike" — The Oxford comma is included before “and.”
 
 ## Ellipses (...)
-Ellipses (...) are commonly used when information is omitted. You might use ellipses when you cannot fit all words onto a line or when you remove less relevant information (like in a quote).
+**Ellipses (...)** are commonly used when information is omitted. Use ellipses when you cannot fit all words onto a line or when you remove less relevant information (like in a quote). For example:
 
 <div class="ws-content-table">
 
-|**Before ellipses**  | **After ellipses** |
+|**Before**  | **After** |
 |---------------------|--------------------|
 | They said, “For many reasons, I think the PatternFly community is great.” | They said, “...I think the PatternFly community is great.” |
 
 </div>
 
-Ellipses can also be used in more creative contexts to signify someone’s thoughts or speech, like a pause for thinking.
-
-Example:
-
-But I was just trying to...never mind, forget it.
+Ellipses can also be used in more creative contexts to signify someone’s thoughts or speech, like a pause for thinking. For example, "But I was just trying to...never mind, forget it."
 
-## Em dash, en dash, hyphen
-Use an **em dash (—)** to add emphasis to the phrase or sentence following it. You can also use an em dash to provide additional information or specification in the middle of a sentence.
+## Em dash, en dash, and hyphen
```

Comment:
Wonder if you should add what each of these are in parentheses, the same way you do with the Ampersand and Ellipses!